grails-app/utils/org/pih/warehouse/LocalizationUtil.groovy

```diff
@@ -148,4 +149,16 @@ class LocalizationUtil {
     static String getDefaultString(String str) {
         return getLocalizedString(str, null)
     }
+
+
+    static boolean isSpanishMexico(Locale locale) {
+        return locale == new Locale("es", "MX")
+    }
```

Comment:
I feel like it should be more generic. Going this path we should create a function for checking every locale we have?

Reply:
I don't expect methods for all of the locales in the near future. I think for now this method is self-explanatory and it's not a good time to overthink it, is it?  
If we wanted to do it generic it would be probably as simple as passing an additional param like `localeToCompare`, but the question is, if we feel strong that we should do it this way?